main_test.go

```diff
@@ -47,10 +47,11 @@ func TestCLI(t *testing.T) {
 	// Validate new list output
 	c.Run(t, TestCase{
 		Command: "list",
-		StdOut: `
-NAME          ADDRESS                     NAMESPACE    WEB                                                 STATUS    
-localhost     localhost:7233              default                                                          
-production    temporal.example.com:443    myapp        http://localhost:8080/namespaces/myapp/workflows    active`,
+		StdOutContains: []string{
+			"NAME          ADDRESS                     NAMESPACE    WEB                                                 STATUS",
+			"localhost     localhost:7233              default",
+			"production    temporal.example.com:443    myapp        http://localhost:8080/namespaces/myapp/workflows    active",
+		},
```

Comment:
Only thing I changed here was to use `StdOutContains`, as the previous version was highly susceptible to editor formatting changes.

Technically the test is now more accepting (if it printed a third line, or garbage between them, it would still pass), but I think that's a small risk. An alternative unit test could be written that *just* checks the data that the `list` command actually pulls.